devops/developer-api/openapi.yml

```diff
@@ -0,0 +1,1843 @@
+          example: "us-east-1"
+        read_regions:
+          type: array
+          items:
+            type: string
+            enum:
+              - us-east-1
+              - us-west-1
+              - us-west-2
+              - eu-west-1
+              - eu-central-1
+              - ap-southeast-1
+              - ap-southeast-2
+              - ap-northeast-1
+              - sa-east-1
+          description: Array of Read Regions of the Database
+          example: ["us-west-1", "us-west-2"]
+        plan:
+          type: string
+          description: |
+            Specifies the fixed plan type for the database.
+            If omitted, the database defaults to either the pay-as-you-go or free plan, based on the account type.
+          enum:
+            - payg
+            - fixed_250mb
+            - fixed_1gb
+            - fixed_5gb
+            - fixed_10gb
+            - fixed_50gb
+            - fixed_100gb
+            - fixed_500gb
+          example: "payg"
+        budget:
+          type: integer
+          description: Monthly budget of the database
+          example: 100
+      required:
+        - name
+        - region
+        - primary_region
+
+    ChangePlanRequest:
+      type: object
+      properties:
+        plan_name:
+          type: string
+          description: The new plan for the database
+          enum:
+            - payg
+            - fixed_250mb
+            - fixed_1gb
+            - fixed_5gb
+            - fixed_10gb
+            - fixed_50gb
+            - fixed_100gb
+            - fixed_500gb
+          example: "fixed_1gb"
+        auto_upgrade:
+          type: boolean
+          description: Whether to enable automatic upgrade for the database
+          example: true
+        prod_pack_enabled:
+          type: boolean
+          description: Whether to enable the production pack for the database
+          example: false
+      required:
+        - plan_name
+    DatabaseStats:
+      type: object
+      properties:
+        connection_count:
+          type: array
+          items:
+            $ref: "#/components/schemas/TimeSeriesData"
+          description: Connection count over time
+        keyspace:
+          type: array
+          items:
+            $ref: "#/components/schemas/TimeSeriesData"
+          description: Total number of keys in the database over time
+        throughput:
+          type: array
+          items:
+            $ref: "#/components/schemas/TimeSeriesData"
+          description: Throughput on database connections over time
+        produce_throughput:
+          type: array
+          nullable: true
+          items:
+            $ref: "#/components/schemas/TimeSeriesData"
+          description: Write throughput over time
+        consume_throughput:
+          type: array
+          nullable: true
+          items:
+            $ref: "#/components/schemas/TimeSeriesData"
+          description: Read throughput over time
```

Comment:
let's just remove the ones listed here from the docs

https://upstash.slack.com/archives/C099375LDKM/p1756716293035659